scripts/clean-cargo-tests.sh

```diff
@@ -21,6 +21,9 @@ done
 # Delete the directory too
 nix-sudo rmdir --ignore-fail-on-non-empty "/tmp/io-engine-tests" 2>/dev/null
 
+# If there was a soft rdma device created and left undeleted by nvmf rdma test,
+# delete that now. Not removing rdma-rxe kernel module.
+sudo rdma link delete rxe0 2>/dev/null
```

Comment:
Don't use sudo directly.
Btw maybe we can use name io-engine-rxe0? (or can we not dictate the name?)

```suggestion
nix-sudo rdma link delete rxe0 2>/dev/null
```

Reply:
name is arbitrary, we can use `io-engine-rxe0`

Reply:
renamed to `io-engine-rxe0`